### ISSUE TITLE
Usunięcie powiążania konta z aplikacja allegro

### ISSUE THREAD

**Issue by konradkur** [allegro/allegro-api#5687]
Witam,
czy jest endpoint, ktory usunie powiazania naszego konta z aplikacja z developer.allegro.pl?
Konrad

**Comment by AureliuszBrussy** [comment]
Nie, nie udostępniamy takiego endpointu. Powiązanie można usunąć tylko z poziomu zakładki [Powiązane aplikacje](https://allegro.pl/moje-allegro/moje-konto/powiazane-aplikacje).
